index.html

```diff
@@ -1121,6 +1121,8 @@ <h3>Contexts</h3>
   ]</span>,
   "id": "http://example.edu/credentials/58473",
   "type": ["VerifiableCredential", "AlumniCredential"],
+  "issuer": "https://example.edu/issuers/565049",
+  "issuanceDate": "2010-01-01T19:23:24Z",
```

Comment:
I'd love if some of the `issuanceDate` value example to show that less precision is acceptable:
```suggestion
  "issuanceDate": "2010-01-01T00:00:00Z",
```

Reply:
Sure, I see no problem with that.